doc/appendix-fr-boi-tva-decl-30-10-30/data-structures/data-structures.md

```diff
@@ -4,7 +4,27 @@ This chapter expands on the descriptions of the data structures covered in the C
 
 ### Receipt Request
 
-There are no special requirements or laws for the French market.
+This table describes fields of the Receipt Request applicable to the French market.
+
+| Field name                | Data type | Default Value Mandatory Field | Description                                                                                              | Version |
+|---------------------------|-----------|-------------------------------|----------------------------------------------------------------------------------------------------------|---------|
+| cbReceiptAmount	        |Decimal?	| null<br>mandatory	                | Total receipt amount incl. taxes (gross receipt amount).                                                 | 0-      |
+
+#### ReceiptCaseData Entry
+
+ReceiptCaseData Entry is used to hand over additional required Informations to a data line for a receipt.
+The Format is limited to JSON.
+For French law fulfullment a  ManagerId is required, if the receipt is sent as training receipt.
+
+| **Field Name** | **Data Type** | **Default Value Mandatory Field** | **Description**                                                       | **Version** |
+|----------------|---------------|-----------------------------------|-----------------------------------------------------------------------|-------------|
+| `ManagerId`	                |String<br>Max 1k |	empty-string<br>optional<br>mandatory in training mode | Identification of the user, who started the training mode. | 0-      |
+
+##### ReceiptCaseData Entry Example
+
+`{
+   \"ManagerId\" : 17
```

Comment:
ManagerId is defined as a string above, but here it's a number. Let's be consistent, even if JSON parses could probably live with this :)